webviz_subsurface/_datainput/pvt_data.py

```diff
@@ -152,124 +154,190 @@ def load_pvt_dataframe(
     drop_ensemble_duplicates: bool = False,
 ) -> pd.DataFrame:
     # pylint: disable=too-many-statements
-    # If ecl2df is not loaded, this machine is probably not
-    # running Linux and the modules are not available.
-    # To avoid a crash, return an empty DataFrame here.
-    if "ecl2df" not in sys.modules:
-        print(
-            "Your operating system does not support opening and reading"
-            " Eclipse files. An empty data frame will be returned and your"
-            " plot will therefore not show any data points. Please specify"
-            " a relative path to a PVT CSV file in your PvtPlot settings"
-            " to display PVT data anyways."
-        )
-        return pd.DataFrame({})
 
     def ecl2df_pvt_data_frame(kwargs: Any) -> pd.DataFrame:
+        # If ecl2df is not loaded, this machine is probably not
+        # running Linux and the modules are not available.
+        # To avoid a crash, return an empty DataFrame here.
+        if "ecl2df" not in sys.modules:
+            raise ModuleNotFoundError(
+                "Your operating system does not support opening and reading"
+                " Eclipse files. An empty data frame will be returned and your"
+                " plot will therefore not show any data points. Please specify"
+                " a relative path to a PVT CSV file in your PvtPlot settings"
+                " to display PVT data anyways."
+            )
```

Comment:
Comments are not valid anymore (not returning empty dataframe to avoid crash) + suggest to install ecl2df if on Linux (think maybe these lines are too long now). Maybe also gather the two ecl2df checks to one check function for less maintenance?
```suggestion
        # If ecl2df is not loaded, this machine is probably not
        # running Linux and the modules are not available, or ecl2df
        # is not installed (not in setup.py).
        if "ecl2df" not in sys.modules:
            raise ModuleNotFoundError(
                "`ecl2df` is not installed. If you run on Linux, try `pip install ecl2df`."
                " Otherwise your operating system does not support opening and"
                " reading Eclipse files. Please specify a relative path to a PVT CSV"
                "  file in your PvtPlot settings to display PVT data anyways."
            )
```